compiler/passes/scopeResolve.cpp

```diff
@@ -1405,8 +1405,15 @@ static void resolveModuleCall(CallExpr* call) {
 
         currModule->moduleUseAdd(mod);
 
+        Symbol* sym;
+
         // First, try regular scope resolution
-        Symbol* sym = scope->lookupNameLocally(mbrName);
+        sym = scope->lookupNameLocally(mbrName);
```

Comment:
Why did you break this onto a separate line?

Reply:
I.... don't know?